zerver/models.py

```diff
@@ -140,7 +140,7 @@ class Realm(models.Model):
 
     # User-visible display name and description used on e.g. the organization homepage
     name = models.CharField(max_length=MAX_REALM_NAME_LENGTH, null=True)  # type: Optional[str]
-    description = models.TextField(default=u"")  # type: str
+    description = models.TextField(default=settings.DEFAULT_ORGANIZATION_DESCRIPTION)  # type: str
```

Comment:
As noted below, we shouldn't do this model change; the previous value of `""` was perfectly clear about the organization description being unset.